### ISSUE TITLE
Error on startup

### ISSUE THREAD

**Issue by EtzBetz** [jsclayton/prometheus-plex-exporter#18]
Hello, when running this container via docker-compose in portainer, on a Raspberry Pi 4, I only get this error as output:

`standard_init_linux.go:219: exec user process caused: exec format error`

My docker-compose for the container looks like this:

```docker
---
version: '3.8'

services:
  prom-plex-exporter:
    image: ghcr.io/jsclayton/prometheus-plex-exporter
    ports:
      - 9001:9000/tcp
    environment:
      PLEX_SERVER: <Plex Server URL> (yes i changed this on the deployment)
      PLEX_TOKEN: <Plex Admin Token> (yes i changed this on the deployment)
```

Is the image compatible with raspberry pi? Or do you see any other error here?

**Comment by jzucker2** [comment]
It doesn't seem like this builds anything but default images, which is likely `amd64`

If it says 32 bit, you

**Comment by EtzBetz** [comment]
> It doesn't seem like this builds anything but default images, which is likely `amd64`
> 
> If it says 32 bit, you

what do you mean by the 2nd sentence?

**Comment by craigvphillips** [comment]
> Is the image compatible with raspberry pi? Or do you see any other error here?

No. I assume you have this fixed by now? I had to build the go binary on my pi, and then create a new docker image with that ARM binary.

**Comment by EtzBetz** [comment]
> > Is the image compatible with raspberry pi? Or do you see any other error here?
> 
> No. I assume you have this fixed by now? I had to build the go binary on my pi, and then create a new docker image with that ARM binary. 
> 
> 

No, I haven't yet. Wasn't a big priority.
